src/components/Button/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Button } from 'antd';
+const ButtonComponent = ({ btnName, type }: { btnName: string; type: any }) => {
```

Comment:
1. `btnName` is not a good name, should use `label` or `btnText`
2. should use `const ButtonComp: React.FC<IButton> = ({...}) => { ...}`. See `src/ExampleComponent`
3. Don't use `any`. In your editor, move mouse pointer to type in `Button` of `antd`, see the small window popup, you can know it type.

Reply:
Fixed!